src/main/java/io/snyk/jenkins/SnykReportBuildAction.java

```diff
@@ -48,7 +48,7 @@ public String getUrlName() {
 
   @SuppressWarnings("unused")
   public String getArtifactContent() {
-    String filename = Stapler.getCurrentRequest().getParameter("artifact");
+    String filename = Stapler.getCurrentRequest2().getParameter("artifact");
```

Comment:
Jenkins 2.479.x introduces Jakarta EE9. The former method was deprecated and replaced by `getCurrentRequest2`.